marinerescue-frontend/src/pages/AboutPage.js

```diff
@@ -1,11 +1,121 @@
+                                    <h3>PM/Designer</h3>
+                                </div>                            
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="row">
+                        <div class="column">
+                            <h1>- Acknowledgements - </h1>
+                            <h2>Our team is grateful for the support from our mentors and peers at the University of Washington’s Information School for their advice through each iteration of our project. Throughout the development of this project, we have worked with the Coastal Observation and Seabird Survey Team (COASST). The team at COASST has helped ensure the validity of our content and provided materials such as photos and informational handouts from their Marine Debris program which are incorporated in our educational activities. Our team would like to thank COASST for their incredible support through this development process.  </h2>
+                            <br />
+                            <h2>COASST is a citizen science program housed at the University of Washington with more than 20 years of history. The program’s goal is to focus on beach environment, specifically marine debris and beached birds in the northeast Pacific area. To learn more about COASST, visit their <a href="https://coasst.org/">website</a>.</h2>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="footer">
+                    <div class="row">
+                            <div class="column">                        
+                            </div>
+                            <div class="column">
+                                <h3>Connect with us: </h3>
+                                <img id="icon" src="images/youtube-logo.png"></img> <br />
+                            </div>
+                            <div class="column">
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="column">
+                                <div class="navbar-logo">
+                                    <img src="images/logo.png"></img>
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <h3>@2021 AppleJACK</h3>
+                            </div>
+                            <div class="column">
+                                <img id="icon" src="images/ischool-logo.png"></img> <br />
+                                <img id="icon" src="images/coasst-logo.png"></img>
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
Can you please move this to be its own component <Footer> so we can re-use it everyone on our website? Thanks!